integration/tests/crud/creategetdelete_test.go

```diff
@@ -924,7 +924,7 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 				"--timeout=45m",
 				"--cluster", params.ClusterName,
 				"--nodes", "1",
-				"--instance-types", "p3.2xlarge,p3.8xlarge,g3s.xlarge,g4ad.xlarge,g4ad.2xlarge",
+				"--instance-types", "g6.xlarge,g6.2xlarge",
```

Comment:
should work but might run into insufficient capacity with this type FYI 

if we go a bit larger to 4x, 8x, 16x - the price won't be too drastic given its run a short duration but there will be more available capacity (customers avoid these because they really need the GPUs, not the added vCPU/memory - so they all aim for the x, 2x size) 
<img width="454" alt="image" src="https://github.com/user-attachments/assets/82bed505-bc82-4c4e-8f3e-e0032ccc48fe" />

Reply:
Yeah. I wanted to try using another instance type entirely, but they're either also old or gigantic (like the P5 instances).

We can try this for now, and then if there are issues adjust